dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessTaskRelationServiceImpl.java

```diff
@@ -165,6 +174,84 @@ public Map<String, Object> createProcessTaskRelation(User loginUser, long projec
         return result;
     }
 
+    private ProcessTaskRelationLog syncObj2Log(User user, ProcessTaskRelation processTaskRelation) {
```

Comment:
i am here use key word `persist` instead of `convert`, the keyword of this method is to persist `task relation log` into database  instead of covert to `task relation log`, the reason we return  `task relation log` it is for convenient, and we should renturn some thing when we created a new record